docs/src/man/comparisons.md

```diff
@@ -235,6 +235,58 @@ The table below compares more advanced commands:
 | DataFrame as input        | `summarize(df, head(across(), 2))`                        | `combine(d -> first(d, 2), df)`                               |
 | DataFrame as output       | `summarize(df, tibble(value = c(min(x), max(x))))`        | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
 
+## Comparison with the R package data.table
+
+The following table compares the main functions of DataFrames.jl with the R package data.table (version 1.14.1). To contrast data.table with dplyr, the focus has been put on transformations in place.
```

Comment:
"To contrast data.table with dplyr" is a bit of a weird goal here: the point of comparisons with other implementations is to allow their users to understand DataFrames.jl easily, not to compare them with other implementations. Does data.table support non-in-place operations?

Reply:
I just nixed it. I am assuming the guide is for people coming over from data.table rather than the opposite. so they know what they generally know that data.table has lots of operations that are changes in place.